packages/lesswrong/server/callbacks/localgroupCallbacks.ts

```diff
@@ -14,3 +16,9 @@ getCollectionHooks("Localgroups").updateValidate.add((validationErrors: Array<an
   
   return validationErrors;
 });
+
+getCollectionHooks("Localgroups").updateAfter.add(async (newDoc: DbLocalgroup, {oldDocument}: {oldDocument: DbLocalgroup}) => {
+  const newOrganizerIds = difference(newDoc.organizerIds, oldDocument.organizerIds)
+  await createNotifications({userIds: newOrganizerIds, notificationType: "newGroupOrganizer", documentType: "localgroup", documentId: newDoc._id})
+  return newDoc
+})
```

Comment:
What about after create?

Reply:
Thanks for the comments! Guess this PR was a bit sloppy :P